src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'parser/core/Analyzer';
+import StatisticBox from 'interface/others/StatisticBox';
+import ItemHealingDone from 'interface/others/ItemHealingDone';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber } from 'common/format';
+
+
+
+/**
+ * Simple info graphic that shows what spell did what healing during wotc window, the hps of each spell, and hptc
+ */
+
+class WayOfTheCrane extends Analyzer {
+
+  customMap = null;
+  damageSpell = "";
+  lastTimeStamp = 0;
+  inWotc = false;
+  wotcTime = 0;
+
+  constructor(...args) {
+    super(...args);
+    const hasEssence = this.selectedCombatant.hasEssence(SPELLS.CONFLICT.traitId);
+    if(!hasEssence){
+        return;
+    }
+    this.active = this.selectedCombatant.hasMajor(SPELLS.CONFLICT.traitId);
```

Comment:
are both of these checks necessary? (legit not sure)

Reply:
Pretty sure. The essence object will bug out and die if you try to check if an essence is major if it doesn't exist. And wotc is the conflict (major) part of conflict and strife. So 99% sure

Reply:
👍